src/devtools/client/debugger/src/components/SecondaryPanes/Frames/Group.js

```diff
@@ -133,7 +133,7 @@ export default class Group extends Component {
     const l10NEntry = this.state.expanded
       ? "callStack.group.collapseTooltip"
       : "callStack.group.expandTooltip";
-    const title = l10n.getFormatStr(l10NEntry, frame.library);
+    const title = l10n?.getFormatStr(l10NEntry, frame.library);
```

Comment:
I have no idea why this is `undefined` here but not in the `SecondaryPanel`, but not having a title doesn't seem to render strangely 🤷

Reply:
It'd be nice to delete all the references to `l10n?.getFormatStr` we can inline the strings